sql/bit.go

```diff
@@ -0,0 +1,124 @@
+package sql
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cast"
+	"vitess.io/vitess/go/sqltypes"
+	"vitess.io/vitess/go/vt/proto/query"
+)
+
+type BitType interface {
```

Comment:
For all these types, you should have a type comment that explains what the type represents and maybe a link to the MySQL docs for it.